src/components/Media/lib/convertSizesToWidths.test.ts

```diff
@@ -0,0 +1,22 @@
+import { convertSizesToWidths } from './convertSizesToWidths';
+
+describe('convertSizesToWidths', () => {
+    it('Returns empty array when `sizes` does not contain valid size defintions', () => {
+        expect(convertSizesToWidths('')).toEqual([]);
+        expect(convertSizesToWidths('test string 123')).toEqual([]);
+        expect(convertSizesToWidths('test, 123')).toEqual([]);
```

Comment:
Should it maybe throw instead? Just to not cover possible human mistakes?

Reply:
Yeah, I think it makes sense to throw 👍🏻